### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
Unit Test Cases:

- Tested deployment of SP on plain FSR instance, deployed dependency packages "FortiSOAR Essentials Solution Pack"

- Verified if all the Bot commands give correct output as mentioned below, to execute this type any of the below commands as a comment
    1. $Bot --help
        - This command gives all the available Bot Command option list

    2. $Get Alert _Alert ID_
        - This commands gives alert information
            > Alert Name
            > Description
            > Alert Link

    3. $Get Incident _Incident ID_
        - This commands gives Incident information
            > Incident Name
            > Description
            > Incident Link

    4.  $Get Reputation _Indicator Value_
        - This command gives the reputation of the indicator

    5. $Get GeoLocation _IP Address or Domain_
        - This command gives geolocation of provided IP Address or Domain

- Verified that for invalid command input below error shows up
    > Error:
        Invalid Bot command has been provided.
        Please refer to the $Bot --help for more details

- Verified that for failed execution of command below error shows up
    > Error:
        Command execution is failed.
        Please refer to the "$Bot --help" to ensure proper command syntax is given.
        Also, refer to the playbook execution history for more details on the error

### CODE REVIEW

File: playbooks/tags.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[
+    "PostCreate",
+    "system",
```
Comment (cs-suyog-jinturkar) [inline playbooks/tags.json]: Why these are marked as system?

Comment (cs-rohit-khune) [reply playbooks/tags.json]: These playbooks were already marked as System. Do this tag need to be removed? @cs-suyog-jinturkar

Comment (cs-suyog-jinturkar) [reply playbooks/tags.json]: yes

Comment (cs-suyog-jinturkar) [reply playbooks/tags.json]: @cs-rohit-khune

Comment (cs-rohit-khune) [reply playbooks/tags.json]: @cs-suyog-jinturkar As suggested, removed 'system' tags from all the playbooks. 
Please review

**Pull request merged**